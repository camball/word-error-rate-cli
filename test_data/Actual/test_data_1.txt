Agent: Hello these is a testing
Customer: Great! Thanks you.
Agent: This is another lines of test data; how is your today?