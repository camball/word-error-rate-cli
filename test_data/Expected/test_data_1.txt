Agent: Hello this is a test.
Customer: Great! Thank you.
Agent: This is another line of test data; how are you today?